src/capabilities.c

```diff
@@ -206,7 +206,7 @@ static MHD_RESULT cap_http_handler(void *cls, struct MHD_Connection *connection,
    * round. The docs are not very specific on the issue. */
   if (*connection_state == NULL) {
     /* set to a random non-NULL pointer. */
-    *connection_state = &(int){44};
+    *connection_state = (void *)1;
```

Comment:
To me it would be clearer if this used same as `write_prometheus.c` plugin code:
```
    /* keep track of connection state */
    *connection_state = (void *)"called";
```

And the comment above stating "The docs are not very specific on the issue" is not really on-spot either.  MHD docs explicitly state that first call for each connection is after headers have been parsed, and should be used only for reporting errors (like is stated in  `write_prometheus.c`).